packages/core/src/interpreter.ts

```diff
@@ -124,7 +124,13 @@ export class Interpreter<
     },
     logger: global.console.log.bind(console),
     devTools: false
-  }))(typeof window === 'undefined' ? global : window);
+  }))(
+    typeof window !== 'undefined'
```

Comment:
We could just use `self` over `window` as `window.self` is supported in all popular browsers

Reply:
Done